pkg/controller/common/cleaner_test.go

```diff
@@ -237,3 +238,109 @@ func AssertAction(t *testing.T, got []k8stesting.Action, expected k8stesting.Act
 
 	require.True(t, found, "expected to find action", expected)
 }
+
+func TestNewCleaner(t *testing.T) {
+	m, err := manager.New(restConfig, manager.Options{MetricsBindAddress: "0"})
```

Comment:
what's the reason for binding the metrics to 0?

Reply:
`MetricsBindAddress` is the TCP address that the controller should bind to for serving prometheus metrics. It can be set to "0" to disable the metrics serving. I wanted to disable it because I didn't see a need for metrics and also for each `manager.New(...)` I would have to use different port in address, otherwise, I would get error: `error listening on :8080: listen tcp :8080: bind: address already in use`.

Reply:
makes sense!